### ISSUE TITLE
RCS1264 not working for `SyntaxTree[] syntaxTree = new SyntaxTree[sourceText.Length];`

### ISSUE THREAD

**Issue by glen-84** [JosefPihrt/Roslynator#1665]
```ini
# Use 'var' or explicit type.
dotnet_diagnostic.RCS1264.severity = warning
roslynator_use_var = always
```

```cs
SyntaxTree[] syntaxTree = new SyntaxTree[sourceText.Length]; // ❌ Not reported.
```
